core/update.php

```diff
@@ -347,18 +408,18 @@ function update_info_page() {
 function update_access_denied_page() {
   backdrop_add_http_header('Status', '403 Forbidden');
   watchdog('access denied', 'update.php', NULL, WATCHDOG_WARNING);
-  backdrop_set_title(t('Access denied'));
+  backdrop_set_title('Access denied');
 
   $output = '';
   $steps = array();
 
-  $output .= t('You are not authorized to access this page. Log in using either an account with the <em>administer software updates</em> permission, or the site maintenance account (the account you created during installation). If you cannot log in, you will have to edit <code>settings.php</code> to bypass this access check. To do this:');
+  $output .= 'You are not authorized to access this page. Log in using either an account with the <em>administer software updates</em> permission, or the site primary account (the account you created during installation). If you cannot log in, you will have to edit <code>settings.php</code> to bypass this access check. To do that:';
```

Comment:
To reference something just described or mentioned, *this* is correct. *this access check* does not seem correct, as *Log in* is at the beginning of the previous sentence and that is not an *access check*.